automation/scripts/github_branch_autosync/Makefile

```diff
@@ -0,0 +1,48 @@
+clean:
+	find . -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete
```

Comment:
Using `find` to delete files feels a bit dangerous to me.

Would it be safer to delete files with `rm`, if their location is known?

Reply:
ok, but "widely-used" doesn't mean "good" :-)